perfmetrics/scripts/continuous_test/gcp_ubuntu/build.sh

```diff
@@ -57,7 +57,8 @@ chmod +x run_load_test_and_fetch_metrics.sh
 
 sudo umount $MOUNT_POINT
 
-# ls_metrics test. This test does gcsfuse mount first and then do the testing.
+# ls_metrics test. This test does gcsfuse mount with the passed flags first and then does the testing.
+GCSFUSE_FLAGS_SUBSET="--implicit-dirs --max-conns-per-host 100 --enable-storage-client-library"
```

Comment:
Since, the difference in flags passed for fio and list tests is just the log file, so we can try to reuse the flags variable, so that future changes have to be done at one place only.
Can we do something like:
GCSFUSE_FLAGS="--implicit-dirs --max-conns-per-host 100 --enable-storage-client-library --debug_fuse --debug_gcs  --log-format \"text\" --stackdriver-export-interval=30s"
GCSFUSE_FIO_FLAGS=GCSFUSE_FLAGS + "--log-file $LOG_FIO_FILE"

Similarly, for list test
GCSFUSE_LIST_FLAGS=GCSFUSE_FLAGS + "--log-file $LOG_LIST_FILE"